server/views/temporary-accommodation/premises/index.njk

```diff
@@ -2,16 +2,16 @@
 {% from "govuk/components/table/macro.njk" import govukTable %}
 {% extends "../../partials/layout.njk" %}
 
-{% set pageTitle = applicationName + " - Properties" %}
+{% set pageTitle = applicationName + " - List of properties" %}
```

Comment:
Completely up to you but we have been passing through the pageHeading from the controller as a variable for methods we've written recently.
I find it helps me work out which view we're rendering without having to look in the template

Reply:
Cool, cheers for the heads up